core/src/OpenACC/Kokkos_OpenACC_ParallelReduce_Range.hpp

```diff
@@ -70,10 +70,8 @@ class Kokkos::Impl::ParallelReduce<Functor, Kokkos::RangePolicy<Traits...>,
                                    ReducerType, Kokkos::Experimental::OpenACC> {
   using Policy = RangePolicy<Traits...>;
 
-  using Analysis = FunctorAnalysis<
-      FunctorPatternInterface::REDUCE, Policy,
-      std::conditional_t<std::is_same_v<InvalidType, ReducerType>, Functor,
-                         ReducerType>>;
+  using Analysis =
+      FunctorAnalysis<FunctorPatternInterface::REDUCE, Policy, Functor>;
```

Comment:
Why don't you take the `ReducerType` into account here anymore?

Reply:
Ping

Reply:
Updated to take into account ReducerType